designer/designer_content.py

```diff
@@ -100,10 +100,11 @@ def add_file_to_tree_view(self, _file):
                     break
 
             if not found:
-                for component in list_path_components:
-                    _node = TreeViewLabel(text=component)
-                    self.tree_view.add_node(_node, node)
-                    node = _node
+                # Weird path mixup when reloading project from dir changes
+                # for component in list_path_components:
+                #     _node = TreeViewLabel(text=component)
+                #     self.tree_view.add_node(_node, node)
+                #     node = _node
```

Comment:
Please don't comment out portions of the code that are not relevant to your pr,  if there is a issue separate from you pr then we need a separate fix.